### ISSUE TITLE
Dependency problem during Quickstart

### ISSUE THREAD

**Issue by bugeats** [reasonml/reason-react#652]
I just sat down to try ReasonReact for the first time. The ReasonReact homepage currently has this quickstart copy pasta:

    npm install -g bs-platform
    bsb -init my-react-app -theme react-hooks
    cd my-react-app
    npm install && npm start

However, `npm install` is currently failing with:

```
ERESOLVE unable to resolve dependency tree

While resolving: <my-new-project>@0.1.0
Found: bs-platform@9.0.2
node_modules/bs-platform
  dev bs-platform@"^9.0.2" from the root project

Could not resolve dependency:
peer bs-platform@"^7.1.1" from reason-react@0.9.1
node_modules/reason-react
  reason-react@">=0.7.1" from the root project
```

Which is kinda a bad look, ya know?

**Comment by bugeats** [comment]
As of today, here's the deps that I could get to resolve and then boot the server:
```json
    "dependencies": {
        "react": "^16.8.1",
        "react-dom": "^16.8.1",
        "reason-react": ">=0.7.1"
    },
    "devDependencies": {
        "bs-platform": "^7.1.1",
        "moduleserve": "^0.9.0"
    }
```

The key thing is that I changed `bs-platform` to `^7.1.1`.

**Comment by stevematdavies** [comment]
This is not good indeed, there now appears to be several ways to theoretically get a Reason React app up, none work, and in fact, bucklescript is deprecated, so what gives?

**Comment by mishaszu** [comment]
`reason-react`: `^0.9.1` has the same issue. It throws an error on installation.
In my case, if I force npm install. It throws an error but completes installation and app work anyway.

**Comment by davesnx** [comment]
Released a new version (0.10.0) that should fix any installation issue with latest bs-platform (v9). Let me know if you find any issues and will try to fix them. 

Thanks for reporting those and sorry for the late response/fix.
